src/relay/backend/aot_executor_codegen.cc

```diff
@@ -45,6 +45,13 @@ namespace tvm {
 namespace relay {
 namespace backend {
 
+struct EnumClassHash {
```

Comment:
Could you EnumClassHash into "utils.h" and remove this definition of it and the definition in the te compiler?

Reply:
done